### PR TITLE
[doc] Document scroll service

### PR DESCRIPTION
**Pull request by drapisarda**
https://homeday.atlassian.net/browse/FET-103

### CODE REVIEW

File: src/services/README.md [hunk 1]
```diff
@@ -219,3 +219,49 @@ const amount = 3;
...
+```
+
+### isWindowFullyScrolled
+Returns a boolean indicating if the document is scrolled until the very bottom.
+```javascript
+const theDocumentIsScrolled = isWindowFullyScrolled(); // false
+```
+
+### freezeScrolling/unfreezeScrolling
+Those functions encapsulates `disableBodyScroll` from [*willmcpo's body-scroll-lock*]((https://github.com/willmcpo/body-scroll-lock)) . Them allows to freeze/unfreeze the document scroll, keeping it allowed on the target element passed as argument.
```
Comment (ilyasmez) [inline src/services/README.md:247]:
```suggestion
These functions encapsulate `disableBodyScroll` from [*willmcpo's body-scroll-lock*]((https://github.com/willmcpo/body-scroll-lock)) which supports iOS browsers. They allow freezing/unfreezing the document scroll, keeping it allowed on the target element passed as argument.
```

File: src/services/README.md [hunk 2]
```diff
@@ -219,3 +219,49 @@ const amount = 3;
...
+
+`param {Object} el: the DOM element to check`
+`param {Number} [offset=window.innerHeight]: the value to check the bottom of the element. It's default value has been chose to check, as default, if the element is scrolled past the bottom of the window.`
+
+### scrollToEl
+Scrolls the window to the top of a given element. It accepts as an argument an **object** and uses three properties:
+
+`param {Object} el: the DOM element to scroll to`
+`param {Number} [offset=0]: scrolling offset to the element.`
+`param {Boolean} [onlyIfNotVisible=true]: scrolls to the element only if this is not visible. The check of the visibility is made with isElVisible described above`
```
Comment (ilyasmez) [inline src/services/README.md:267]:
```suggestion
`param {Boolean} [onlyIfNotVisible=true]: scrolls to the element only if it's is not visible in the viewport. The check of the visibility is made with isElVisible described above`
```

File: src/services/README.md [hunk 3]
```diff
@@ -219,3 +219,49 @@ const amount = 3;
 const randomInts = generateUniqueNumbers(amount, min, max);
 // [2, 3, 5]
 ```
+
+
+## Scrolling
+We provide a collection of fuctions to manage elements position and the page scrolling.
```
Comment (ilyasmez) [inline src/services/README.md:225]:
```suggestion
We provide a collection of functions to manage elements position and the page scrolling.
```

**Review by ilyasmez — COMMENTED** [review]
🚀 🤓 
I've just fixed some spelling mistakes and added iOS browsers for the scroll freezing definition as I think it's worth mentioning.

**Comment by ilyasmez** [comment]
It will also look better if you add line breaks between the params of `isElVisible`, `isScrolledPastElBottom` and `scrollToEl`.

**Review by volcanioo — APPROVED** [review]

File: src/services/README.md (hunk 2 above)
Comment (drapisarda) [reply src/services/README.md:267]: Gold

**Pull request merged**